ios/Runner/AppDelegate.swift

```diff
@@ -11,6 +12,41 @@ import WebKit
         var views = [String: WKWebView]()
 
         let controller: FlutterViewController = window?.rootViewController as! FlutterViewController
+        let ttsChannel = FlutterMethodChannel(name: "tts_plugin", binaryMessenger: controller.binaryMessenger)
+        ttsChannel.setMethodCallHandler {
+            (call: FlutterMethodCall, result: @escaping FlutterResult) in
+            let args = call.arguments as? [String: Any]
+            let synthesizer = AVSpeechSynthesizer()
+
+            switch call.method {
+            case "speak":
+                guard let text = args?["text"] as? String, !text.isEmpty else {
```

Comment:
Empty text is ok, actually. If the text is empty return `result(Bool(true))` immediately.